hck_setup.cfg

```diff
@@ -19,6 +19,7 @@ HCK_ROOT=`dirname $0`
 
 #QEMU PATH
 QEMU_BIN=/usr/libexec/qemu-kvm
+QEMU_IMG_BIN=/usr/libexec/qemu-img
```

Comment:
Let's keep as default the old value

Reply:
Sorry, I'm not sure that I understand this one: ```QEMU_IMG_BIN``` was not defined before, and use was made of a hard-coded ```qemu-img```. So what do you mean by "old value"?